scripts/dnu.sh

```diff
@@ -8,5 +8,7 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
+# work around restore timeouts on Mono
+if [[ "$@" == "restore"* ]]; then export MONO_THREADS_PER_CPU=50; fi
```

Comment:
@davidfowl yeah, I did it this way to avoid impacting other commands like build, but I guess it doesn't really matter. Changed it.